src/App.vue

```diff
@@ -1,26 +1,27 @@
+
 <template>
-  <img alt="Vue logo" src="./assets/logo.png">
-  <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <div class="flex flex-col justify-center items-center ">
+        <GoStop @start-game="startGame" @stop-game="stopGame" />
```

Comment:
@guptasakshi282001 same mistake again? What I aked you in your last Assignment about component naming?